### PR TITLE
Fixes #24352: Plugin information is too empty when there is no plugin

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/24352

### CODE REVIEW

**Comment by ElaadF** [comment]
Commit modified

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/plugins/pluginInformation.html [hunk 1]
```diff
@@ -30,6 +30,16 @@ <h1>Plugins</h1>
         </div>
         <div class="main-details">
           <div class="lift:administration.PluginManagement.display">
+            <div class="rudder-package-doc callout-fade callout-info">
+              <div class="marker"><span class="glyphicon glyphicon-info-sign"></span></div>
+                <p>Plugins are managed by the rudder package manager.</p>
+                <p>
+                  You can consult the list of plugin with the command <b>rudder package list --all</b><br />
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/webapp/secure/plugins/pluginInformation.html:37]:
```suggestion
                  You can consult the list of plugins with the command <b>rudder package list --all</b><br />
```

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/plugins/pluginInformation.html [hunk 2]
```diff
@@ -30,6 +30,16 @@ <h1>Plugins</h1>
...
+                  To consult the list of plugin already installed you can use the command <b>rudder package list</b><br />
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/webapp/secure/plugins/pluginInformation.html:38]:
```suggestion
                  To consult the list of plugins already installed you can use the command <b>rudder package list</b><br />
```

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/plugins/pluginInformation.html [hunk 3]
```diff
@@ -30,6 +30,16 @@ <h1>Plugins</h1>
...
+                  Finally you can install a plugin with the command <b>rudder package install plugin-name</b><br />
+                </p>
+                <p>Rudder package manager comes with more command, to consult the help use the command <b>rudder package help</b></p>
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/webapp/secure/plugins/pluginInformation.html:41]:
```suggestion
                <p>Rudder package manager comes with more commands, to consult the help use the command <b>rudder package help</b></p>
```

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/plugins/pluginInformation.html [hunk 4]
```diff
@@ -30,6 +30,16 @@ <h1>Plugins</h1>
         </div>
         <div class="main-details">
           <div class="lift:administration.PluginManagement.display">
+            <div class="rudder-package-doc callout-fade callout-info">
+              <div class="marker"><span class="glyphicon glyphicon-info-sign"></span></div>
+                <p>Plugins are managed by the rudder package manager.</p>
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/webapp/secure/plugins/pluginInformation.html:35]:
```suggestion
                <p>Plugins are managed by the Rudder package manager.</p>
```

**Comment by ElaadF** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by ElaadF** [comment]
OK, squash merging this PR

**Pull request merged**